net/open-isns/Makefile

```diff
@@ -0,0 +1,55 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=open-isns
+PKG_VERSION:=0.100
+PKG_RELEASE:=1
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/open-iscsi/open-isns/archive/
+PKG_HASH:=b011edbb0f31690aaca902a8ecf4e1f17b01d6c9e9afc51909d26b0993b4328f
+
+PKG_MAINTAINER:=Lucian CRISTIAN <lucian.cristian@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/open-isns
+	SECTION:=libs
+	CATEGORY:=Libraries
+	URL:=http://www.open-iscsi.com/
+	TITLE:=open-isns
+	DEPENDS:=+libopenssl
+endef
+
+define Package/open-isns/description
+This is a partial implementation of iSNS, according to RFC4171.
+endef
+
+CONFIGURE_ARGS += \
+    --prefix=/usr \
```

Comment:
replace spaces with TAB.

Only --enable-shared is needed.

Reply:
Not needed
https://github.com/openwrt/openwrt/blob/master/include/package-defaults.mk#L80
LIB_DIR isn't defined so I have no opinion about that one.